backend/tournesol/admin.py

```diff
@@ -84,6 +84,11 @@ class EntityAdmin(admin.ModelAdmin):
     )
     actions = ["update_metadata"]
 
+    def get_queryset(self, request):
+        qst = super().get_queryset(request)
+        qst = qst.with_prefetched_poll_ratings(poll_name="videos")
```

Comment:
We saw with @aidanjungo that this method may not be appropriate here.

It makes the related `EntityPollRating` object available in a list, which prevent us to easily order the entities by Tournesol score or number of comparisons.

```python
    @admin.display(description="comparisons", ordering="???")
    def get_n_comparisons(obj):
        return obj.single_poll_ratings[0].n_comparisons
```

We may need to retrieve the Tournesol score, n comparisons and n contributors in a different way (don't forget to check the generated SQL query and the performance of the ordering).